### PR TITLE
New hosts UI features

### PR DESCRIPTION
**Pull request by Levi-Leah**
Add the procedures on how to use new Hosts UI

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_adding-a-host-to-host-collections.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_adding-a-host-to-host-collections.adoc:10]:
```suggestion
. In the *Host collections* card, click the vertical ellipsis and select *Add host to collections*.
```

File: guides/common/modules/proc_adding-a-host-to-the-host-group.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_adding-a-host-to-the-host-group.adoc:3]:
```suggestion
[id="adding-a-host-to-a-host-group_{context}"]
= Adding a Host to a Host Group
```

File: guides/common/modules/proc_adding-a-host-to-the-host-group.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_adding-a-host-to-the-host-group.adoc:5]:
```suggestion
You can add a host to a host group in the {ProjectWebUI}.
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:5]:
```suggestion
Use this procedure to change the content source for a host.
```
Please also update title, anchor, and xrefs accordingly.

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:9]:
```suggestion
. Click the name of the host you want to modify.
```

File: guides/common/modules/proc_changing-the-host-group-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-host-group-of-a-host.adoc:17]:
```suggestion
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:10]:
```suggestion
. Click the vertical ellipsis icon next to the *Edit* button and select *Change content source*.
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:11]:
```suggestion
. Select Environment, Content View, and Content Source from the lists.
```

File: guides/common/modules/proc_changing-the-environment-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-environment-of-a-host.adoc:8]:
```suggestion
. Click the name of the host you want to modify.
```

File: guides/common/modules/proc_changing-the-environment-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-environment-of-a-host.adoc:9]:
```suggestion
. Click the vertical ellipsis in the *Content view details* card and select *Edit content view assignment*.
```

File: guides/common/modules/proc_changing-the-host-group-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-host-group-of-a-host.adoc:10]:
```suggestion
. Click the name of the host you want to modify.
```

File: guides/common/modules/proc_changing-the-host-group-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-host-group-of-a-host.adoc:12]:
```suggestion
. Select the new host group from the *Host Group* list.
```

File: guides/common/modules/proc_changing-the-host-group-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-host-group-of-a-host.adoc:16]:
```suggestion
* The *Details* card under the *Overview* tab now shows the host group your host belongs to.
```

File: guides/common/modules/proc_adding-a-host-to-the-host-group.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_adding-a-host-to-the-host-group.adoc:15]:
```suggestion
* The *Details* card under the *Overview* tab now shows the host group your host belongs to.
```

File: guides/common/modules/proc_changing-the-managed-status-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-managed-status-of-a-host.adoc:13]:
```suggestion
. Click the name of the host you want to modify.
```

File: guides/common/modules/proc_changing-the-module-stream-for-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-module-stream-for-a-host.adoc:10]:
```suggestion
. Click the name of the host you want to modify.
```

File: guides/common/modules/proc_changing-the-module-stream-for-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_changing-the-module-stream-for-a-host.adoc:14]:
```suggestion
You get a REX job notification once the remote execution job is complete.
```

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:15]:
```suggestion
. Click the name of the host you want to modify.
```

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:16]:
```suggestion
. Click *Edit* on the *System purpose* card.
```

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:17]:
```suggestion
. Select the system purpose attributes for your host.
```

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:19]: I don't know the answer to this question so you will either have to tag someone specifically or ask an SME.

Comment (Levi-Leah) [reply guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:19]: > I don't know the answer to this question so you will either have to tag someone specifically or ask an SME.

Yeah, I’m adding a few procedure and tagging SME + QA. Thanks for the feedback!

File: guides/common/modules/proc_adding-a-host-to-host-collections.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_adding-a-host-to-host-collections.adoc:15]: Is the "Verification" section new? cc @Lennonka @adamlazik1

Comment (Levi-Leah) [reply guides/common/modules/proc_adding-a-host-to-host-collections.adoc:15]: @maximiliankolb It's part of the standard CCS template. It's been there for quite some time. The section is used in other files in this repo too guides/common/modules/proc_enabling-capsule-in-UI.adoc:

File: guides/common/modules/proc_restarting-applications-on-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_restarting-applications-on-a-host.adoc:4]:
```suggestion
[id="restarting-applications-on-a-host_{context}"]
= Restarting Applications on a Host
```

File: guides/common/modules/proc_enabling-tracer-on-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_enabling-tracer-on-a-host.adoc:4]:
```suggestion
[id="enabling-tracer-on-a-host_{context}"]
= Enabling Tracer on a Host
```

File: guides/common/modules/proc_restarting-applications-on-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_restarting-applications-on-a-host.adoc:12]:
```suggestion
. Select applications that you want to restart.
```

File: guides/common/modules/proc_restarting-applications-on-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_restarting-applications-on-a-host.adoc:13]:
```suggestion
. Select *Restart via remote execution* from the *Restart app* list.
```

File: guides/common/modules/proc_restarting-applications-on-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_restarting-applications-on-a-host.adoc:15]:
```suggestion
You will get a REX job notification once the remote execution job is complete.
```

File: guides/common/modules/proc_selecting-the-upgrade-version-of-a-package.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_selecting-the-upgrade-version-of-a-package.adoc]:
```suggestion
[id="selecting-the-upgrade-version-of-a-package_{context}"]
= Selecting the Upgrade Version of a Package
```

File: guides/common/modules/proc_adding-a-host-to-host-collections.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-a-host-to-host-collections.adoc:15]: I am not against this at all. It's just that I haven't seen this before. Thanks for explaining.

What is CCS? Are the templates public?

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (jeremylenz) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:5]: Auto-attach is only relevant when SCA is turned off.

I would suggest using the text Rich Jerrido gave me for the tooltip:
_System purpose allows you to set the system's intended use on your network and improves the reporting of usage in Subscription Watch._

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (jeremylenz) [reply guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:19]: No, there isn't an auto-attach button here, because most users will eventually be using SCA and not need it. Users who need to run auto-attach will need to visit the legacy UI. See also my comment above.

File: guides/common/modules/proc_removing-packages-from-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_removing-packages-from-a-host.adoc:3]:
```suggestion
[id="removing-packages-from-a-host_{context}"]
= Removing Packages from a Host
```

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc
Comment (jeremylenz) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:19]: Why remove this section? Did it move somewhere else?

File: guides/common/modules/proc_removing-packages-from-a-host.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_removing-packages-from-a-host.adoc:11]:
```suggestion
. Click the vertical ellipsis for the package you want to remove and select *Remove*.
```

**Review by jeremylenz — COMMENTED** [review]
Thanks for looping me in! Left some comments below but mostly looks great.

File: guides/common/modules/proc_upgrading-packages-on-a-host-in-in-bulk.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_upgrading-packages-on-a-host-in-in-bulk.adoc:3]:
```suggestion
[id="upgrading-packages-on-a-host-in-bulk_{context}"]
= Upgrading Packages on a Host in Bulk
```

File: guides/common/modules/proc_upgrading-packages-on-a-host-in-in-bulk.adoc
Comment (adamlazik1) [inline guides/common/modules/proc_upgrading-packages-on-a-host-in-in-bulk.adoc:11]:
```suggestion
. Select *Upgradable* from the *Status* list.
```

File: guides/doc-Managing_Content/master.adoc
Comment (adamlazik1) [inline guides/doc-Managing_Content/master.adoc:42]:
```suggestion
include::common/modules/proc_upgrading-packages-on-a-host-in-bulk.adoc[leveloffset=+1]
```

**Review by adamlazik1 — COMMENTED** [review]
Please note that we use Camel-Case styled headings unlike newdoc which uses sentence-case style headings, so it is necessary to update the heading after creating a module.

**Comment by Levi-Leah** [comment]
> it is necessary to update the heading after creating a module.

Aye

**Comment by Levi-Leah** [comment]
> it is necessary to update the heading after creating a module.

Aye

**Comment by Levi-Leah** [comment]
Asked Shweta for review

File: guides/common/modules/proc_enabling-tracer-on-a-host.adoc
Comment (jeremylenz) [inline guides/common/modules/proc_enabling-tracer-on-a-host.adoc:12]:
```suggestion
* {project-client-name} repository is available in the host's content view and lifecycle environment
```

File: guides/common/assembly_administering-hosts.adoc
Comment (Lennonka) [inline guides/common/assembly_administering-hosts.adoc:25]:
```suggestion
ifdef::katello,satellite,orcharhino[]
include::modules/proc_adding-a-host-to-a-host-collection.adoc[leveloffset=+1]
endif::[]
```
Host collections are available only in products that include Katello.
(We've discussed this with @ekohl in [#1711](https://github.com/theforeman/foreman-documentation/pull/1711/files#r999411719))
Also pls unify the wording of the filename and ID.

File: guides/common/assembly_administering-hosts.adoc
Comment (Lennonka) [inline guides/common/assembly_administering-hosts.adoc:21]:
```suggestion
include::modules/proc_adding-a-host-to-a-host-group.adoc[leveloffset=+1]
```
Unify wording of the filename and ID.

File: guides/common/modules/proc_changing-the-content-source-of-a-host.adoc
Comment (Lennonka) [inline guides/common/modules/proc_changing-the-content-source-of-a-host.adoc:17]:
```suggestion
To update the content source manually, execute the autogenerated commands from *Change content source* on the host.
```

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (Lennonka) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:16]:
```suggestion
. On the *Overview* tab, click *Edit* on the *System purpose* card.
```

File: guides/common/modules/proc_enabling-tracer-on-a-host.adoc
Comment (Lennonka) [inline guides/common/modules/proc_enabling-tracer-on-a-host.adoc:3]:
```suggestion
[id="enabling-tracer-on-a-host_{context}"]
```

**Review by Lennonka — CHANGES_REQUESTED** [review]
A couple of remarks, but looks good mostly!

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc
Comment (jeremylenz) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:5]: Per Jamie Prevatt, this text should now change to
```suggestion
System purpose enables you to set the system's intended use on your network and improves reporting accuracy in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

(sorry for the multiple edits! ping me if you want more context :)

File: guides/common/assembly_administering-hosts.adoc
Comment (Levi-Leah) [reply guides/common/assembly_administering-hosts.adoc]: > 

good to know

**Comment by Lennonka** [comment]
I see an empty file: `modules/proc_editing-the-system-purpose-of-multiple-hosts.adoc` 
If you're not going to fill it, please remove it from the PR.

File: guides/common/assembly_administering-hosts.adoc
Comment (Lennonka) [inline guides/common/assembly_administering-hosts.adoc:26]:
```suggestion
include::modules/proc_adding-a-host-to-a-host-collection.adoc[leveloffset=+1]
```

I tried to rename the file, but there already is a file of this name. Pls, clean up.

**Comment by Lennonka** [comment]
Just FYI, this should probably go to 3.4 as well. I'm adding a tick for it.

**Review by Lennonka — APPROVED** [review]
LGTM, once the checks finish, I'm merging it.

**Pull request merged**
